tests/effective_dps_test.cpp

```diff
@@ -444,17 +444,17 @@ static void check_two_handed_swords( const std::function<Approx( const std::stri
 {
     SECTION( "two-handed swords" ) { // expected value around 27, 25 for long swords
         CHECK( calc_expected_dps( "nodachi" ) == 26.5 );
-        CHECK( calc_expected_dps( "zweihander" ) == 27.0 );
+        CHECK( calc_expected_dps( "zweihander" ) == 23.79 );
```

Comment:
It would be reasonable to increase the damage to get this back up a bit.

Reply:
Will do, they got a bit more clumsy so I can up the damage a few points.  They should still shred regardless.